bitmovin-analytics-datasource/src/types.ts

```diff
@@ -18,3 +18,18 @@ export interface MyDataSourceOptions extends DataSourceJsonData {
   tenantOrgId?: string;
   adAnalytics?: boolean;
 }
+
+export enum QUERY_INTERVAL {
+  SECOND = 'SECOND',
+  MINUTE = 'MINUTE',
+  HOUR = 'HOUR',
+  DAY = 'DAY',
+  MONTH = 'MONTH',
+  AUTO = 'AUTO',
+}
```

Comment:
tbh I would replace enum with simple 
`export type QueryInterval = 'SECOND' | 'MINUTE' | 'HOUR' | 'DAY' | 'MONTH' | 'AUTO';`, because at the end , it is "just a string" that we send as `interval` in request payload, and we just want typescript to protect its type.

Reply:
I would also move that `type` together with `intervalToMilliseconds` utility into one file, group code together, like `utils/queryIntervalUtils.ts` , because based on experience from Dashboard, you will need maybe more such utilities doing something with query interval